Taskfile.yml

```
@@ -328,6 +319,43 @@ tasks:
           URL_PREFIX: "{{.NODEJS_VERSION_BASE_URL}}"
           OUTPUT_DIR: "{{.OUTPUT_DIR}}"
 
+  package-tar:
+    internal: true
+    requires:
+      vars: ["FLAVOUR", "STORAGE_ENGINE"]
+    vars:
+      VERSIONED_PACKAGE_NAME:
+        sh: |
+          source /etc/os-release
+          echo "clp-{{.FLAVOUR}}-${ID}-${VERSION_CODENAME}-$(arch)-v{{.G_PACKAGE_VERSION}}"
+      OUTPUT_DIR: "{{.G_BUILD_DIR}}/{{.VERSIONED_PACKAGE_NAME}}"
+      OUTPUT_FILE: "{{.G_BUILD_DIR}}/{{.VERSIONED_PACKAGE_NAME}}.tar.gz"
```

Is there a reason we don't form `OUTPUT_FILE` from `OUTPUT_DIR`? i.e.,
```suggestion
      OUTPUT_FILE: "{{.OUTPUT_DIR}}.tar.gz"
```
I'm fine keep those separated if they mean to differentiate the semantics.

---

This is from before we confirmed that `task` respects variable declaration order.